### ISSUE TITLE
Unclear usage of `asyncapi start`

### ISSUE THREAD

**Issue by Lazzaretti** [asyncapi/cli#1090]
#### Describe the bug

To me it is unclear how you use `asyncapi start`.

#### How to Reproduce

1. Run command `asyncapi`
2. you see the command explanation
   `start     Start asyncapi studio`
3. Run command `asyncapi start`
4. You get a help text.

```
Start asyncapi studio

USAGE
  $ asyncapi start

DESCRIPTION
  Start asyncapi studio

COMMANDS
  start studio  starts a new local instance of Studio
````

#### Expected behavior

"Start asyncapi studio" - as described in the help text.

Probably related to https://github.com/asyncapi/cli/issues/551

**Comment by peter-rr** [comment]
Hi @Lazzaretti 
Totally agree with the unclear usage of `asyncapi start` 👍 . Actually this is gonna be included as part of new UI/UX improvements for the CLI: https://github.com/asyncapi/cli/issues/1214.

Could you please remove the "bug" label since we should consider this issue as an improvement or enhancement instead?
Thank you for reporting :)

**Comment by Lazzaretti** [comment]
Hi @peter-rr, cool that you want to change this :)

Unfortunately, I can't remove the bug label; I think this can only be done by a contributor (my name is only there because on creation, you can select "bug").

**Comment by peter-rr** [comment]
@Lazzaretti 
I guess that can only be changed by the repository maintainers. Let me ping them.

@derberg @Souvikns @magicmatatjahu 
Could you change this issue's label from `bug` to `enhancement`? Thanks!

**Comment by AayushSaini101** [comment]
@Souvikns @peter-rr I am bit confused here, do we need to start studio instead of generating this text.Start asyncapi studio
? 
```
USAGE
  $ asyncapi start

DESCRIPTION
  Start asyncapi studio

COMMANDS
  start studio  starts a new local instance of Studio
```

**Comment by peter-rr** [comment]
@AayushSaini101 According to how it's implemented the `start` command currently, when we run `asyncapi start` we're just executing the `index.ts` [subcommand](https://github.com/asyncapi/cli/blob/master/src/commands/start/index.ts), showing the "help" section. To start a new studio instance we need to run: `asyncapi start studio -f specFile_path`.

I agree is not very DX friendly, though 🤔

**Comment by AayushSaini101** [comment]
> @AayushSaini101 According to how it's implemented the `start` command currently, when we run `asyncapi start` we're just executing the `index.ts` [subcommand](https://github.com/asyncapi/cli/blob/master/src/commands/start/index.ts), showing the "help" section. To start a new studio instance we need to run: `asyncapi start studio -f specFile_path`.
> 
> I agree it's not very DX friendly, though 🤔

@peter-rr  @Amzani I am willing to work on it

**Comment by AayushSaini101** [comment]
> @AayushSaini101 According to how it's implemented the `start` command currently, when we run `asyncapi start` we're just executing the `index.ts` [subcommand](https://github.com/asyncapi/cli/blob/master/src/commands/start/index.ts), showing the "help" section. To start a new studio instance we need to run: `asyncapi start studio -f <specFile_path>`.
> 
> I agree it's not very DX friendly, though 🤔

@peter-rr Why we cannot remove the asyncapi start command and use only asyncapi start studio, because the description of the asyncapi start is  Start asyncapi studio🤔

**Comment by peter-rr** [comment]
> @peter-rr Why we cannot remove the asyncapi start command and use only asyncapi start studio, because the description of the asyncapi start is Start asyncapi studio🤔

Yeah, that was something we already discussed on https://github.com/asyncapi/cli/issues/551.
I guess the `asyncapi start` command was designed originally with other purposes, to "start" different processes like `studio`, for instance. That's why `index.ts` is located under `start` folder. But no more subcommands have been added so far, so I don't think this makes sense anymore.

I believe we should rid off this `index.ts` file and just keep `studio.ts`. Also we could rename the command as simply `asyncapi studio`, which would be more DX friendly from my point of view. Any thoughts on that? @Souvikns @Shurtu-gal @Amzani

**Comment by Amzani** [comment]
We can just change the help text @peter-rr, 
As mentioned https://github.com/asyncapi/cli?tab=readme-ov-file#command-structure-and-patterns this command might evolve in the future depending on:
- Do we add more sub-commands to studio, for example `synchronization` then it will make sense to add a studio noun, and then we will have: `asyncapi studio start` and `asyncapi studio sync`
- Do we add an API as mentioned in https://github.com/asyncapi/cli/pull/1200, then we will have `asyncapi start api`
